### PR TITLE
dailyjournal todaystart configurable templatename

### PR DESCRIPTION
**Pull request by m1well**
* template name for `/todayStart` and `/dayStart` is now configurable 
* i also added a hint in the README that one has to be careful with `/dayStart` on another day than today with tags like `date... / formattedDate...` and `weather()`

### CODE REVIEW

File: jgclark.DailyJournal/src/journal.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //--------------------------------------------------------------------------------------------------------------------
 // Daily Journal plugin for NotePlan
 // Jonathan Clark
-// last update v0.9.0, 25.11.2021 by @jgclark/@m1well
+// last update v0.10.0, 07.12.2021 by @jgclark/@m1well
```
Comment (jgclark) [inline jgclark.DailyJournal/src/journal.js:5]: Last update is just you not me as well!

File: jgclark.DailyJournal/src/journal.js [hunk 2]
```diff
@@ -54,6 +56,29 @@ export async function dayStart(today: boolean = false): Promise<void> {
...
+  {
+    console.log('\tWarning: Cannot find suitable \'dailyJournal\' settings in Templates/_configuration note. Stopping.')
+    await showMessage(
+      'Cannot find \'dailyJournal\' settings in _configuration.',
+      'Yes, I\'ll check my _configuration settings.',
+    )
+    return
+  }
+
+  pref_templateTitle = (journalConfig?.nameOfDailyNoteTemplate != null)
```
Comment (jgclark) [inline jgclark.DailyJournal/src/journal.js:78]: Using two very different names for the same thing is confusing.
Can I suggest you use `templateTitle` not `nameOfDailyNoteTemplate` which better fits with the style of other settings, and matches the `pref_templateTitle` in the code?

**Review by jgclark — CHANGES_REQUESTED** [review]
A helpful addition, thanks.
I'm just requesting a couple of stylistic changes. And an update to the CHANGELOG.

**Comment by m1well** [comment]
done @jgclark

**Review by jgclark — APPROVED** [review]

**Pull request merged**